### ISSUE TITLE
Component does not clean up correctly when A-Frame inspector loads

### ISSUE THREAD

**Issue by HAZARDU5** [wizgrav/aframe-effects#4]
I've found that when opening the inspector, the effects layer doesn't seem to be removed from the scene, preventing interaction with entities in the inspector viewport. See the official demo here for an example of this: https://wizgrav.github.io/aframe-effects/index.html

In the demo, unchecking 'Toggle Effects' before opening the inspector allows the inspector viewport to function as expected.

**Comment by wizgrav** [comment]
I've debugged a bit and It seems strange, even if I start the inspector without post processing and then do eg. a scene.setAttribute("effects", "bloom, fxaa") the post processing will take effect only after you revert to the non inspector scene.

My guess is that another scene is created and all components are serialized to it for the inspector. The effects attribute updates the system though. I'll need to consult @fernandojsg on this

**Comment by wizgrav** [comment]
Ok I think I'm getting somewhere, scene.isPlaying is false so tick and tocks are not getting called it's not a new scene it's just that the scene is set up to render on scene.renderTarget (when set rendering will happen to that instead of directly on the canvas). I need to find a way to get notified when play state changes

**Comment by diegodorado** [comment]
ANy update o workaround on this issue?

**Comment by wizgrav** [comment]
Yeah this needs some update, I was planning to do it along with the ssao effect which also is almost done but then life and changing countries and job put it on halt. I'll try to at least fix this and some more minor issues this w/e
